### PR TITLE
Remove README instructions to add addon to ADDONS_REQUESTED [OSF-7089]

### PR DESCRIPTION
**Pull request by alexschiller**
## Purpose

ADDONS_REQUESTED was removed from local.py, but addons still referenced ADDONS_REQUESTED. Adding ADDONS_REQUESTED to local.py breaks addons. 

## Changes
Remove ADDONS_REQUESTED references from addon READMEs.

## Side effects

None.

## Ticket

https://openscience.atlassian.net/browse/OSF-7089

### CODE REVIEW

**Comment by brianjgeiger** [comment]
ping @mfraezz

File: website/addons/dropbox/README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 Enabling the addon for development
 
-1. In `website/settings/local.py` add, `"dropbox"` to `ADDONS_REQUESTED`.
-2. Create a local dropbox settings file with `cp website/addons/dropbox/settings/local-dist.py website/addons/dropbox/setings/local.py`
-3. Create an app and get a key and secret from https://www.dropbox.com/developers/apps.
-5. At the Dropbox app console, add http://localhost:5000/oauth/callback/dropbox/ to your list of Oauth2 redirect URIs.
```
Comment (mfraezz) [inline website/addons/dropbox/README.md]: >2.
>3.
>5.

:+1:

File: website/addons/figshare/README.md [hunk 2]
```diff
@@ -18,4 +18,3 @@ akin to *{"error": "You cannot request an access token for yourself!"}*
   1. Copy the *consumer_key* to **CLIENT_ID**
   2. Copy the *consumer_secret* to **CLIENT_SECRET**
 12. Open website/settings/local.py
```
Comment (mfraezz) [inline website/addons/figshare/README.md]: This should probably be updated/removed too, as just opening this file probably isn't necessary. Also, this file will be changing dramatically whenever #6167 goes in.

**Comment by brianjgeiger** [comment]
@alexschiller Conflicts need resolving.

**Pull request merged**